lib/cc/analyzer/container.rb

```diff
@@ -0,0 +1,98 @@
+require "posix/spawn"
+
+module CC
+  module Analyzer
+    class Container
+      DEFAULT_TIMEOUT = 15 * 60 # 15m
+
+      def initialize(
+        image:,
+        name:,
+        command: nil,
+        log: NullContainerLog.new,
+        timeout: DEFAULT_TIMEOUT
+      )
+        @image = image
```

Comment:
Sorry, got my former and latters mixed up. Editing my original comment to hopefully clarify...